src/views/WafRules/EditView.vue

```diff
@@ -0,0 +1,354 @@
+            :loadService="handleLoadWafRulesAllowedService"
+            :editService="handleEditWafRulesAllowedService"
+            :schema="validationSchemaAllowed"
+            @onSuccess="reloadWafRulesAllowedList"
+            title="Edit Waf Rules Allowed"
+          >
+            <template #formFields>
+              <FormFieldsAllowed />
+            </template>
+          </EditDrawerBlock>
+        </TabPanel>
+      </TabView>
+      <router-view></router-view>
+    </template>
+  </ContentBlock>
+</template>
+<script setup>
+  import Illustration from '@/assets/svg/illustration-layers.vue'
+  import ActionBarTemplate from '@/templates/action-bar-block/action-bar-with-teleport'
+  import ContentBlock from '@/templates/content-block'
+  import EmptyResultsBlock from '@/templates/empty-results-block'
+  import { columnBuilder } from '@/templates/list-table-block/columns/column-builder'
+  import CreateDrawerBlock from '@templates/create-drawer-block'
+  import EditDrawerBlock from '@templates/edit-drawer-block'
+  import EditFormBlock from '@templates/edit-form-block'
+  import ListTableNoHeaderBlock from '@templates/list-table-block/no-header'
+  import PageHeadingBlock from '@templates/page-heading-block'
+  import PrimeButton from 'primevue/button'
+  import TabPanel from 'primevue/tabpanel'
+  import TabView from 'primevue/tabview'
+
+  import { computed, onBeforeMount, ref, watch } from 'vue'
+  import { useRoute, useRouter } from 'vue-router'
+  import * as yup from 'yup'
+  import FormFieldsWafRules from './FormFields/FormFieldsWafRules.vue'
+  import FormFieldsAllowed from './FormFields/FormFieldsAllowed.vue'
+
+  const route = useRoute()
+  const router = useRouter()
+  const activeTab = ref(0)
+  const hasContentToList = ref(true)
+  const selectedWafRulesAllowedToEdit = ref(0)
+  const showEditWafRulesAllowedDrawer = ref(false)
+  const showCreateWafRulesAllowedDrawer = ref(false)
+  const listAllowedRef = ref('')
+
+  const props = defineProps({
+    editWafRulesService: {
+      type: Function,
+      required: true
+    },
+    loadWafRulesService: {
+      type: Function,
+      required: true
+    },
+    listWafRulesAllowedService: {
+      type: Function,
+      required: true
+    },
+    deleteWafRulesAllowedService: {
+      type: Function,
+      required: true
+    },
+    createWafRulesAllowedService: {
+      type: Function,
+      required: true
+    },
+    loadWafRulesAllowedService: {
+      type: Function,
+      required: true
+    },
+    editWafRulesAllowedService: {
+      type: Function,
+      required: true
+    },
+    documentationServiceAllowed: {
+      required: true,
+      type: Function
+    },
+    documentationServiceTuning: {
+      required: true,
+      type: Function
+    }
+  })
+
+  const validationSchemaAllowed = yup.object({
+    matchZones: yup.array(),
+    path: yup.string().required(),
+    reason: yup.string().required(),
+    ruleId: yup.string().required(),
```

Comment:
Recomendo adicionar a label nesses campos required:
```suggestion
    ruleId: yup.string().required().label('rule id'),
```